.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn test
```

Comment:
Do we really need a pre push hook when we have CI?
How long would local tests take?

Reply:
That's a good question. I just moved it from the package.json as it was already defined. If you think we can get rid of this, I am ok with it.

Reply:
https://github.com/mermaid-js/mermaid/blob/89eea0536d946d31c033043f31eb3213fbdda4b0/.github/workflows/build.yml#L38-L40

As the CI will catch it anyway, running in dev machine before every push seems redundant.

@knsv @ashishjain0512 thoughts?

Reply:
I am open for either way...

It is a bit annoying when you push a PR just to have it fail for some silly reason... However... it also annoying having to run the tests when pushing something totally unrelated to the code.

Skip?

Reply:
Done 😊!